app/javascript/styles/mastodon/containers.scss

```diff
@@ -1,4 +1,4 @@
-.container {
+.container-alt {
```

Comment:
I don't get this changes 🤔 Is it to avoid colision with other usages of `.container` in other templates?